src/library/autodj/autodjprocessor.cpp

```diff
@@ -1276,12 +1278,12 @@ void AutoDJProcessor::useFixedFadeTime(DeckAttributes* pFromDeck,
         double transitionTime = math_min(toDeckOutroStart - startPoint,
                 m_transitionTime);
 
-        pFromDeck->fadeBeginPos = endPoint - transitionTime;
+        pFromDeck->fadeBeginPos = math_max(endPoint - transitionTime, fromDeckPosition);
         pFromDeck->fadeEndPos = endPoint;
         pToDeck->startPos = startPoint;
     } else {
         pFromDeck->fadeBeginPos = endPoint;
-        pFromDeck->fadeEndPos = endPoint - m_transitionTime;
+        pFromDeck->fadeEndPos = endPoint;
```

Comment:
This breaks fading with a negative transition time. AutoDJ stops in this case.

Reply:
This was causes elsewhere, but fixed now.